modules/prediction/submodules/predictor_submodule.cc

```diff
@@ -73,6 +76,10 @@ bool PredictorSubmodule::Proc(
   predictor_writer_->Write(
       std::make_shared<PredictionObstacles>(prediction_obstacles));
 
+  double end_time = apollo::cyber::Time::Now().ToNanosecond();
+  double start_time = submodule_output->frame_start_time();
+  ADEBUG << "End to end time = " << (end_time - start_time) / 1e6 << " ms";
```

Comment:
Use high level timer API instead of hard-code the calculation, with magic numbers like `1e6`, `1e9`.
```
const auto start_time = absl::Now();
...
const auto end_time = absl::Now();
ADEBUG << "End to end time = " << absl::ToDoubleMilliseconds(end_time - start_time) << " ms";
```